model/project_ref.go

```diff
@@ -983,6 +982,7 @@ func mergeBranchAndRepoSettings(pRef *ProjectRef, repoRef *RepoRef) (*ProjectRef
 	defer func() {
 		err = recovery.HandlePanicWithError(recover(), err, "project and repo structures do not match")
 	}()
+	// Don't merge enabled field
```

Comment:
did we mean to leave this comment in?

Reply:
I thought it might be nice to see it explicitly stated but seems redundant so removing

Reply:
yeah there are a few fields that aren't merged so it may be unclear